python/src/otel/otel_sdk/otel-handler

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ef -o pipefail
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+: <<'END_DOCUMENTATION'
+`otel-handler`
+
+This script is a wrapper around the existing `otel-instrument` script to maintain
+backward compatibility while providing a more descriptive name.
+
+It simply forwards all arguments to the `otel-instrument` script.
+
+Usage
+-----
+Use this script exactly as you would use `otel-instrument`:
+
+.. code::
+
+    AWS_LAMBDA_EXEC_WRAPPER = /opt/otel-handler
```

Comment:
Just noting out loud that this looks like upstream Python lambda layer has changed its `AWS_LAMBDA_EXEC_WRAPPER` requirement, but [ours is still the same](https://documentation.solarwinds.com/en/success_center/observability/content/configure/services/python/aws-lambda-instrumentation-python.htm).

Reply:
It just synced with the upstream. Code is not in use I believe.